src/mmw/apps/modeling/mapshed/calcs.py

```diff
@@ -431,28 +437,39 @@ def f_to_c(f):
 
     year_range = endyear - begyear + 1
     stations = tuple([w.station for w in ws])
-    temps = [[[0] * 31 for m in range(12)] for y in range(year_range)]
-    prcps = [[[0] * 31 for m in range(12)] for y in range(year_range)]
+    temps = {s: [[[0] * 31 for m in range(12)]
+                 for y in range(year_range)]
+             for s in stations}
+    prcps = {s: [[[0] * 31 for m in range(12)]
+                 for y in range(year_range)]
+             for s in stations}
 
     with connection.cursor() as cursor:
         cursor.execute(temp_sql, [stations, begyear, endyear])
         for row in cursor.fetchall():
-            year = int(row[0]) - begyear
-            month = int(row[1]) - 1
+            station = int(row[0])
+            year = int(row[1]) - begyear
+            month = int(row[2]) - 1
             for day in range(31):
-                temps[year][month][day] = f_to_c(float(row[day + 2]))
+                temps[station][year][month][day] = f_to_c(float(row[day + 3]))
 
     with connection.cursor() as cursor:
         cursor.execute(prcp_sql, [stations, begyear, endyear])
         for row in cursor.fetchall():
-            year = int(row[0]) - begyear
-            month = int(row[1]) - 1
+            station = int(row[0])
+            year = int(row[1]) - begyear
+            month = int(row[2]) - 1
             for day in range(31):
-                prcps[year][month][day] = float(row[day + 2]) * CM_PER_INCH
-
+                prcp = float(row[day + 3]) * CM_PER_INCH
+                prcps[station][year][month][day] = prcp
```

Comment:
This is resulting in different precipitation results than before. Temp is the same, but I don't think we've ever averaged the precipitation. It's throwing the whole model off.

Reply:
This surfaced a little while ago, last note is that @ajrobbins is going to follow up to determine if it's appropriate to average or not. See https://github.com/WikiWatershed/model-my-watershed/issues/2561 for previous discussion.

Reply:
That's great!! I'll leave it then.